software/controller/lib/hal/stepper.h

```diff
@@ -119,48 +115,46 @@ enum class StepMtrErr {
   InvalidState,  // Invalid state for command.
 };
 
-enum class StepMoveStatus {
-  Stopped,
-  Accelerating,
-  Decelerating,
-  ConstantSpeed,
+enum class MoveStatus {
+  Stopped = 0,
```

Comment:
If specific values are needed, should this Enum also have the underlying type specificed?